### PR TITLE
Navbar Component & Slight CSS Update

### PR DESCRIPTION
**Pull request by Harukaichii**
Edits:
* Added content to NavBar Component
* Fixed some styles in FooterComponent

navbar

Notes:
* For the svgs I guess the best way to access them is after they've been uploaded but idk if that happened yet so I just used the source code
* I wasn't too meticulous with the spacing of the content bc there's also the MLH logo that needs to go in. I know it doesn't line up with the text but I think the text is too far in ...? ;;

### CODE REVIEW

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -1,18 +1,203 @@
...
...8.3089 23.1609 38.4371 22.6011 38.5441L22.601 38.5441C20.2184 38.9995 19.8472 39.0704 19.8472 39.9915C19.8472 41.1289 24.7992 42.2132 24.7992 42.2132C24.7992 42.2132 23.1087 42.2132 23.1087 42.8607V43.5722L25.7996 43.5516C26.3377 43.5474 27.1116 42.9532 27.0958 42.1717C27.0799 41.3901 23.5422 39.9915 23.5422 39.9915C25.8472 39.8216 28.5743 39.5309 28.5743 38.421C28.5743 37.311 23.7131 36.1853 23.7131 36.1853C29.3209 35.6095 28.5743 34.6248 28.5743 34.6248L28.6357 32.6454C31.1655 31.7125 33.1433 29.0334 33.1433 29.0334C37.5554 22.9864 34.1396 16.6844 34.1396 16.6844C31.0873 11.5972 25.9077 11.183 24.0578 11.5047C23.3851 11.6217 22.9715 11.7388 22.7205 11.8441C18.8225 12.934 16.0124 17.2579 16.0124 20.7552C15.7552 17.6681 17.0085 14.104 21.0825 11.5878C18.8094 11.4257 17.0816 12.2058 15.18 13.9106C12.8413 16.3092 11.7853 23.2846 17.718 31.9016C17.832 32.0672 18.0916 31.9539 18.0537 31.7565C17.3397 28.039 17.611 24.9602 19.5272 22.9007ZM30.8274 29.8785C32.5922 28.2478 34.5034 25.4241 34.3218 22.4023C34.3218 22.4023 34.3982 21.5916 34.2361 20.4319C32.6705 19.0264 30.3575 18.607 28.3291 19.5503C25.6158 20.8122 24.4392 24.0347 25.7011 26.748C26.6506 28.7895 28.71 29.961 30.8274 29.8785Z"
```
Comment (jamiepinheiro) [inline src/components/NavBar.vue]: Mind taking a look at how I was doing the floating island, u can add the svg file into the assets folder :)

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -1,18 +1,203 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80%;
+}
+
+.svg-scale2525 {
+  height: 25px;
+  width: 25px;
+}
```
Comment (jamiepinheiro) [inline src/components/NavBar.vue]: ident  inside <style once

File: src/components/NavBar.vue [hunk 3]
```diff
@@ -1,18 +1,203 @@
 <template>
-  <div class="navbar">NavBar</div>
+  <div class="navbar">
```
Comment (jamiepinheiro) [inline src/components/NavBar.vue]: add lass `d-sm-none` to hide on mobile for now

File: src/components/NavBar.vue (hunk 2 above)
Comment (Harukaichii) [reply src/components/NavBar.vue]: oops lol fixed!

File: src/components/NavBar.vue (hunk 1 above)
Comment (Harukaichii) [reply src/components/NavBar.vue]: yessire, thanks ! I didn't know you can important svgs like that haha. I have moved everything out 👍

**Pull request merged**